packages/edge-config/README.md

```diff
@@ -114,3 +114,42 @@ export const config = { runtime: 'edge' };
 3. Within the module you want to test your local development instance of `@vercel/edge-config`, just link it to the dependencies: `npm link @vercel/edge-config`. Instead of the default one from npm, Node.js will now use your clone of `@vercel/edge-config`!
 
 As always, you can run the tests using: `npm test`
+
+## A note for Vite users
+
+`@vercel/edge-config` reads database credentials from `process.env`. With some tools, process.env is automatically populated from your `.env` file during development (created when you run `vc env pull`), but Vite does not expose `.env` variables on `process.env.`
```

Comment:
```suggestion
`@vercel/edge-config` reads database credentials from the environment variables at `process.env`. In general, `process.env` is automatically populated from your `.env` file during development, which is created when you run `vc env pull`. However, Vite does not expose the `.env` variables at `process.env.`
```

Reply:
Will go forward with this copy with a slight modification: `on` is correct, not `at` -- the environment variables are properties _on_ the `process.env` object, not properties _at_ the `process.env` object. Thanks for the improvements! I'll make sure they get added to all of the READMEs.